src/main.ts

```diff
@@ -60,35 +109,55 @@ async function run() {
 
       const podLabel: string = getPodLabel(projectName);
       const hotfixLabel: string = getHotfixLabel(baseBranch);
-      const storyTypeLabel : string = getStoryTypeLabel(story);
+      const storyTypeLabel: string = getStoryTypeLabel(story);
       const labels: string[] = filterArray([podLabel, hotfixLabel, storyTypeLabel]);
 
       console.log('Project name -> ', projectName);
       console.log('Adding lables -> ', labels);
 
-      const repo: string = repository ? repository.name : '';
-      const { number: prNumber, body: prBody } = pull_request ? pull_request : { number: 0, body: '' };
-
       const labelData: IssuesAddLabelsParams = {
-        owner: organization.login,
-        repo,
-        issue_number: prNumber,
+        ...commonPayload,
         labels,
       };
 
-      const client: github.GitHub = new github.GitHub(GITHUB_TOKEN);
       await addLabels(client, labelData);
 
       if (shouldUpdatePRDescription(prBody)) {
         const prData: PullsUpdateParams = {
-          owner: organization.login,
+          owner,
           repo,
           pull_number: prNumber,
           body: getPrDescription(prBody, story),
         };
         await updatePrDetails(client, prData);
+
+        // add comment for PR title
+        if (SKIP_COMMENTS === 'false') {
+          const prTitleComment: IssuesCreateCommentParams = {
+            ...commonPayload,
+            body: getPrTitleComment(story.name, title),
+          };
+          console.log('Adding comment for the PR title');
+          await addComment(client, prTitleComment);
```

Comment:
verrrry minor but these don't need to be synchronous right?
should we do something like `await Promise.all([])` ?

Reply:
like in terms of 
- title comment
- huge pr comment